### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by ryelo**
# Landing Page Redesign

## Mockups
https://marvelapp.com/98130i6/screen/65851600

## Category 1
Screen Shot 2020-02-07 at 8 24 21 AM

## Category 2
Screen Shot 2020-02-07 at 8 24 29 AM

## Category 3
Screen Shot 2020-02-07 at 8 24 35 AM

## Category 3 with all card states present
Screen Shot 2020-02-07 at 8 24 47 AM

## Dev notes
* This separates the consts into a directory with 3 files, depending on the function
    * Marketing consts
    * Technologies consts
    * Unentitled consts
* Disabling is enabled both top level on the categories as well as on individual applications

## To do
- [x] Fill in content on the cards
- [x] Make sure OUIA is correct
- [x] Verify apps are in the correct place

### CODE REVIEW

**Comment by ryelo** [comment]
Still some touch ups that are happening, but the overall review is ready to go.

File: src/layout/Body.js [hunk 1]
```diff
@@ -1,119 +1,80 @@
...
+
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import './Body.scss';
 
-function isBeta() {
-    return (window.insights.chrome.isBeta() === true ? '/beta/' : '/');
-}
+class Body extends React.Component {
```
[the remark this answers is not in the document]
Comment (karelhala) [reply src/layout/Body.js]: Yeah, you could update react and react-dom to newer version in order to use `hooks` in here for state and such.

File: src/components/images/ansibleAutomation.svg [hunk 2]
```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (ryelo) [reply src/components/images/ansibleAutomation.svg]: I'm not sure yet. Waiting on an answer from brand. This may only be used on this page.

**Comment by ryelo** [comment]
@PanSpagetka @karelhala updated a few more thing, made the different consts files consistent, and switched that class component to a hook

File: src/layout/Body.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryelo) [reply src/layout/Body.js]: Done!

**Comment by tahmidefaz** [comment]
@ryelo are we still hiding stuff between `beta` and `stable` envs? Like for example, `Migration Services` is not in `stable` yet, but I see it there and when I click on it, it just reloads the page.

**Comment by ryelo** [comment]
@tahmidefaz Good catch. Updated!

**Comment by ryelo** [comment]
@tahmidefaz one was a mistake. Removed one of them

**Pull request merged**